docs/conventions.rst

```diff
@@ -0,0 +1,95 @@
+
+AppData files
+-------------
+
+AppData files provide metadata about applications, which is used by application stores (such as Flathub, GNOME Software and KDE Discover). The `Freedesktop AppStream specification <https://www.freedesktop.org/software/appstream/docs/>`_ provides a complete reference for providing AppData.
+
+AppData files should be named with the application ID and the ``.appdata.xml`` file extension, and should be placed in ``/app/share/metainfo/``. For example::
+
+  /app/share/metainfo/org.gnome.Dictionary.appdata.xml
+
+The ``appstream-util validate-relax`` command can be used to check AppData files for errors.
+
+XDG base directories
+--------------------
+
+`XDG base directories <https://standards.freedesktop.org/basedir-spec/basedir-spec-latest.html>`_ are a Freedesktop standard which defines standard locations where user-specific application data and configuration should be stored.
+
+By default, Flatpak sets three XDG base directories that should be used by applications for user-specific storage. These are:
+
+===============  =================================  ======================
+Base directory   Usage                              Default location
+===============  =================================  ======================
+XDG_CONFIG_HOME  User-specific configuration files  ~/.var/<app-id>/config
+XDG_DATA_HOME    User-specific data                 ~/.var/<app-id>/data
+XDG_CACHE_HOME   Non-essential user-specific data   ~/.var/<app-id>/cache
+===============  =================================  ======================
+
+For example, GNOME Dictionary should store user-specific data in::
+
+  ~/.var/org.gnome.Dictionary/data
+
+Note that applications can be configured to use non-default base directory locations (see :doc:`sandbox-permissions`).
+
+Filesystem layout
+-----------------
+
+Each Flatpak sandbox, which is the environment in which an application is run, contains the filesystem of the application's runtime. This follows `standard Linux filesystem conventions <https://en.wikipedia.org/wiki/Filesystem_Hierarchy_Standard>`_. For example, the root of the sandbox contains standard directories, such as:
+
+- ``/bin`` - binaries
+- ``/dev`` - device files
+- ``/lib`` - libraries
+- ``/opt`` - application software packages
+- ``/usr`` - multi-user utilities and applications
+
+In addition to this, each sandbox contains a top-level ``/app`` directory, which is where the application's own files are located.
+
+D-Bus
+-----
+
+`D-Bus <https://www.freedesktop.org/wiki/Software/dbus/>`_ is the standard framework for interprocess communication on Linux desktops. Applications do not always need to use it, and often portals will be a better choice. However, there are a few cases where D-Bus is required, such as when integrating with system-provided media controls, through the `MPRIS specification <https://specifications.freedesktop.org/mpris-spec/latest/>`_.
```

Comment:
This is odd phrasing. Applications themselves often use DBus but they do not always need to talk with third party services..

Reply:
> Applications themselves often use DBus

What are the common things that they use it for?

Reply:
Almost all Gtk3 applications for example use it for single instance support. We had this discussion elsewhere but ownership and usage are very different things like MPRIS is a name you own not one you "use" per-se.

Reply:
Should be fixed now.